src/ServicePulse.Host/app/js/app.constants.js

```diff
@@ -5,7 +5,7 @@
         .constant('showPendingRetry', false)
         .constant('scConfig', {
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_url: 'http://localhost:1234/metrics/'
+            monitoring_urls: ['http://localhost:1234/metrics/', 'http://localhost:1235/metrics', 'http://localhost:1236/metrics']
```

Comment:
This is maintained by the installer and is read only, once the array is writted it's only updated with a full restart. No need for any client-side changes.